backend/medtagger/api/users/service.py

```diff
@@ -55,3 +55,16 @@ def get() -> Any:
         """Get user info."""
         user = get_current_user()
         return user, 200
+
+
+@users_ns.route('/<int:user_id>')
+class SetUserInfo(Resource):
+    """Set user's information (first name and last name)."""
+
+    @staticmethod
+    @login_required
+    @users_ns.doc(security='token')
+    def put(user_id: int) -> Any:
+        """Set user info."""
+        set_user_info(user_id, request.json['firstName'], request.json['lastName'])
```

Comment:
Last thought - users should be able to update only their own information ;)
Right now, anyone can update anyone’s else data!

Please use `get_current_user()` to compare IDs. If they don’t match you should return 403.